infra/app/applicationInsights.bicep

```diff
@@ -0,0 +1,24 @@
+param name string
+param location string = resourceGroup().location
+param tags object = {}
+param keyVaultName string
+
+resource applicationInsights 'Microsoft.Insights/components@2020-02-02' = {
+  name: name
+  location: location
+  tags: tags
+  kind: 'web'
+  properties: {
+    Application_Type: 'web'
+  }
+}
+resource keyVault 'Microsoft.KeyVault/vaults@2022-07-01' existing = {

```

Comment:
Spacing, make sure you format all bicep files correctly.

Reply:
Modified.